### ISSUE TITLE
demo-certs / expired DEFAULT_TENANT_CA certificate?

### ISSUE THREAD

**Issue by boaks** [eclipse/hono#2628]
Trying to test x509 client certificate with a localy deployed hono, I get:

> no valid trust anchors defined for tenant [DEFAULT_TENANT]

in the server's log. What makes me wondering is, that
 
```sh
curl -s -i -X GET "http://${REGISTRY_IP}:28080/v1/tenants/DEFAULT_TENANT" -H  "accept: application/json" -H  "Content-Type: application/json"
HTTP/1.1 200 OK
etag: 839c39df-0f03-4513-89a4-eef05c920e72
content-type: application/json; charset=utf-8
content-length: 411

{"enabled":true,"trusted-ca":[{"id":"fa7d45c4-488a-4c91-9ec9-f9119db89f66","subject-dn":"CN=DEFAULT_TENANT_CA,OU=Hono,O=Eclipse IoT,L=Ottawa,C=CA","public-key":"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAElkwCSPlO563eQb6ONdULAISm2XngGGSoAAz+I1s8zkS9guPUpNKoxeczLtKlObelHqBgIZtRXdrPRgXidGOnmQ==","algorithm":"EC","not-before":"2019-09-18T08:35:40Z","not-after":"2020-09-17T08:35:40Z","auto-provisioning-enabled":false}]}
```

Reports a deprecated tenant certificate. The demo-certs contains a new one, valid until 10.11.2021.

I build with 

```sh
mvn clean install -Pbuild-docker-image,jaeger 
./push_hono_images.sh 1.8.0-SNAPSHOT localhost:32000

HELM_SET="$HELM_SET --set adapters.coap.enabled=true"
HELM_SET="$HELM_SET --set jaegerBackendExample.enabled=true"
HELM_SET="$HELM_SET --set honoImagesTag=1.8.0-SNAPSHOT"
HELM_SET="$HELM_SET --set honoContainerRegistry=localhost:32000"

helm install --dependency-update -n hono $HELM_SET eclipse-hono hono
```

using microk8s.

Is there a way/trick to deploy it with the updated tenant ca certificate?

**Comment by calohmn** [comment]
@boaks That seems to be an issue in the IoT packages Hono Helm chart files.
The `example-tenants.sh` script to add the default tenant (see [here](https://github.com/eclipse/packages/blob/b8a638db33c495cdef93b2461fbeeac86622bfb2/charts/hono/example/example-tenants.sh#L24)) doesn't use the correct values (see [here](https://github.com/eclipse/packages/blob/523f6b98becbb9773353d92d9103a0b14258cbaa/charts/hono/example/certs/trust-anchor.properties#L5)).
Can you adopt the values into the `example-tenants.sh` script and try again (deploying the chart from local sources)?

**Comment by boaks** [comment]
The public key hasn't changed?

**Comment by boaks** [comment]
@calohmn 

Just to mention, the new certificate you point to, is not the one on the master of last week ...

**Comment by calohmn** [comment]
The referenced files above are from the IoT packages repo and are different from the example certificates in the Hono repo.
I can provide a fix in the IoT packages project (along with the update to Hono 1.7.1).

**Comment by calohmn** [comment]
The fix is in the Iot Packages Hono Helm chart 1.7.2 (see https://github.com/eclipse/packages/pull/221).
